lms/djangoapps/course_home_api/progress/tests/test_views.py

```diff
@@ -193,48 +192,54 @@ def test_learner_has_access(self):
         assert not gated_score['learner_has_access']
         assert ungated_score['learner_has_access']
 
-    @patch.dict(settings.FEATURES, {'ASSUME_ZERO_GRADE_IF_ABSENT_FOR_ALL_TESTS': False})
     def test_override_is_visible(self):
-        with persistent_grades_feature_flags(global_flag=True):
```

Comment:
Why remove these items in tests? We should leave these in, but outside of the `with` statement right? We want to make the persistentGrade to be true all the time after this change, right?

Reply:
it's right, persistGrade value is always true now, so persistent_grades_feature_flags was removed in this test.
Also completely removed the [persist_grades_feature_flags](https://github.com/openedx/edx-platform/pull/30978/files#diff-9e7e3de020c6b1214b2023343184790e428ef034deff90acd07f83adc0f54063L15) function.